dashboard/sample_metadata_classifier.py

```diff
@@ -293,11 +294,19 @@ def interpret(project, papers, bits):
                 country="Germany",
                 city="Heidelberg",
                 collection="plasma")
-    elif project in papers["Cebria-Mendoza 2021"]["projects"]:
-        sample, = bits
+    #elif project in papers["Cebria-Mendoza 2021"]["projects"]:
+    #    sample, = bits
+    #    return sample, dict(
+    #            country="Spain",
+    #            city="Valencia",
+    #            collection="plasma")
+    elif project in papers["Belstrøm 2017"]["projects"]:
```

Comment:
I think we should probably stick to Ascii for paper names, and here we're inconsistent.  You have "o" on disk (`papers/Belstrom2017`) but "ø" here.

Reply:
done